src/player.cpp

```diff
@@ -41,6 +41,7 @@
 #include <bitset>
 
 #include <fstream>
+#include <cmath>
```

Comment:
I was only using cmath's pow function for a bit of the soft cap, so it's not necessary anymore.  Will remove.